tests/fixtures.ts

```diff
@@ -75,9 +84,36 @@ export const test = baseTest.extend<TestFixtures, WorkerFixtures>({
       const transport = new StdioClientTransport({
         command: 'node',
         args: [path.join(path.dirname(__filename), '../cli.js'), ...args],
+        env: {
+          ...process.env,
+          IS_UNDER_TEST: '1',
+        },
+        stderr: 'pipe',
       });
       client = new Client({ name: 'test', version: '1.0.0' });
       await client.connect(transport);
+      transport.stderr?.addListener('data', data => {
+        const lines = data.toString().trimEnd().split('\n');
```

Comment:
this swallows ending newlines, which could be weird in debugging. is `trimEnd()` necessary?